src/main/java/it/pagopa/pn/deliverypush/action/utils/CheckAttachmentUtils.java

```diff
@@ -22,15 +25,27 @@ public CheckAttachmentUtils(NotificationReceiverValidator validator, PnSafeStora
     }
     
     public void validateAttachment(NotificationInt notification ) throws PnValidationException {
-        log.debug( "Start check attachment for document" );
-        for(NotificationDocumentInt attachment : notification.getDocuments()) {
-            checkAttachment(attachment);
-        }
-        log.debug( "End check attachment for document" );
+        PnAuditLogBuilder auditLogBuilder = new PnAuditLogBuilder();
+        PnAuditLogEvent logEvent = auditLogBuilder
+                .before(PnAuditLogEventType.AUD_NT_VALID, "Start check attachment for iun={}", notification.getIun() )
+                .iun(notification.getIun())
+                .build();
+        logEvent.log();
+        
+        try {
+            for(NotificationDocumentInt attachment : notification.getDocuments()) {
+                checkAttachment(attachment);
+            }
+            logEvent.generateSuccess().log();
```

Comment:
Questo success va spostato in fondo altrimenti se viene sollevata una eccezione dopo mi trovo si il success che il failure